pipeline-model-definition/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/generator/DirectiveGenerator.java

```diff
@@ -107,10 +107,8 @@ public static String mapToClosure(Map<String,?> args) {
                 result.append(arg.getKey()).append(" ");
                 if (arg.getValue() instanceof Map) {
                     result.append(mapToClosure((Map<String, ?>) arg.getValue()));
-                } else if (arg.getValue() instanceof UninstantiatedDescribable) {
-                    result.append(mapToClosure(((UninstantiatedDescribable) arg.getValue()).getArguments()));
                 } else if (arg.getValue() != null) {
-                    result.append(Snippetizer.object2Groovy(arg.getValue())).append("\n");
+                    result.append(Snippetizer.object2Groovy(arg.getValue(), arg.getValue() instanceof UninstantiatedDescribable)).append("\n");
```

Comment:
So is this changing behavior? If so, the PR title is misleading.

Reply:
Updated